docs/team/loycatherine.adoc

```diff
@@ -45,12 +45,37 @@ The highlight of my part is the progress checker of the packing list in the stat
 *** There were a total of 4 releases, from version 1.1 to 1.4. I contributed to all the versions on GitHub.
 
 ** Enhancements to existing features:
-*** Made the UI nicer by adding icons for all the features.  (Pull requests #33, #34)
-*** Wrote additional tests for existing features to increase the coverage from **% to 92%
+*** Made the UI nicer by adding icons for all the features. (Pull requests https://github.com/AY1920S2-CS2103T-W17-3/main/pull/232[#232] https://github.com/AY1920S2-CS2103T-W17-3/main/pull/235[#235],  https://github.com/AY1920S2-CS2103T-W17-3/main/pull/239[#239], https://github.com/AY1920S2-CS2103T-W17-3/main/pull/330[#330])
+*** Wrote additional tests for existing features to help increase the coverage from 20% to 50% (https://github.com/AY1920S2-CS2103T-W17-3/main/pull/143[#143] Pull requests https://github.com/AY1920S2-CS2103T-W17-3/main/pull/359[#359] https://github.com/AY1920S2-CS2103T-W17-3/main/pull/366[#366])
 
 ** Documentation:
 *** Used more 'you' language in the UserGuide so that it will be more reader-friendly. In addition, I added in pictures so that the reader can have visual cues on how the output should look like for each command.
 
 ** Community:
-*** Reviewed Pull Requests
-*** Reported bugs and offered suggestions for other teams in the class (examples: 1, 2, 3)
+*** Reviewed Pull Requests with non-trivial comments (Pull Request https://github.com/AY1920S2-CS2103T-W17-3/main/pull/363[#363])
+*** Reported bugs and offered suggestions for https://github.com/AY1920S2-CS2103T-W16-3/main[StudyBuddy], another AddressBook3-based application.
+
+== Contributions to the User Guide
+
+|===
+|_Given below are sections I contributed to the User Guide. They showcase my ability to write documentation targeting end-users._
+|===
+
+https://github.com/AY1920S2-CS2103T-W17-3/main/blob/master/docs/UserGuide.adoc#packing-list[Packing List]
```

Comment:
Idk if you wanna do this, but we could instead create tags within the UG/DG and then link it

`include::../UserGuide.adoc[tag=addaccommodation]`

Like this is what plan2travel and most other groups are doing.

Reply:
Then in your user guide, just add the tags in. think it might have been deleted somehow.

`//end::accommodation[]
//tag::rearrangingparameters[]`

Reply:
yeah that's a better idea